libraryCompose/src/main/java/app/futured/donut/library/compose/data/DonutDataset.kt

```diff
@@ -0,0 +1,16 @@
+package app.futured.donut.library.compose.data
+
+import androidx.compose.Model
+import androidx.ui.graphics.Color
+
+/**
+ * Data class representing the single section of the donut containing amount and color.
+ *
+ * @param amount value of this single section. Eg. when the [DonutData.cap] is set to 10f and this is set to 2.5f
+ * then this section will take 25% of the whole donut.
+ * @param color color of this single section
+ */
+@Model data class DonutDataset(
```

Comment:
Tady mi moc nesedí ten název `DonutDataSET`. Tenhle název bych chápal u kompletní kolekce těch dat, nikoli u jednoho prvku těch dat. Dává mi smysl mít něco ve smyslu:
```kotlin
data class DonutDataset(
    val donutItems: List<DonutItem>
)

data class DonutItem(
    val name: String,
    val color: Int,
    val amount: Float
)
```

Každopádně chápu, že toto je analogické řešení k `View` verzi, takže mimo scope tohle PR, ale zvážil bych to třeba jako nové issue.

Reply:
Ako hovoríš, snažil som sa to vytvoriť podobne ako v `View` verzii. Predtým som to mal pomenované ako `ProgressSection`

Reply:
Súhlas, keď sa na to tak s odstupom času pozriem, tak je to trochu nešťastné pomenovanie. Zostalo to kvôli tomu, že `DonutDataset` kedysi obsahoval zoznam Floatov, ktoré sa vo view iba sčítali.